goty-client/src/app/form/list-item/list-item.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="list-item">
+  <div>
+    <span *ngIf="orderable" class="prefix">{{index | indexToWord}}:</span>
+    <span class="title">{{game.title}}</span>
+  </div>
+  <div class="controls">
+    <i *ngIf="orderable" style="font-size: 1.2em" [ngClass]="index === 0 ? 'disabled' : ''" class="pi pi-chevron-up" (click)="moveUp.emit(index)"></i>
+    <i *ngIf="orderable" style="font-size: 1.2em" [ngClass]="index === length - 1 ? 'disabled' : ''" class="pi pi-chevron-down" (click)="moveDown.emit(index)"></i>
+    <i class="pi pi-minus" style="font-size: 1.2em" (click)="delete.emit(index)"></i>
```

Comment:
Why style here instead of in CSS?

Reply:
Because i'm specifying an internal style of the prime icon.I could also have done it in the global styles.scss but that way is a bit easier

Reply:
Actually I was wrong. The above is true for components but I could actually (and did) move the styling to the sheet for prime icons.